pkg/cluster/task/update_dm_meta.go

```diff
@@ -13,65 +13,63 @@
 
 package task
 
-/*
 import (
 	"fmt"
 	"strings"
 
-	"github.com/pingcap/tiup/pkg/dms/meta"
+	dmspec "github.com/pingcap/tiup/components/dm/spec"
 
 	"github.com/pingcap/tiup/pkg/set"
 )
 
 // UpdateDMMeta is used to maintain the DM meta information
 type UpdateDMMeta struct {
 	cluster        string
-	metadata       *meta.DMMeta
+	metadata       *dmspec.DMMeta
```

Comment:
Can we rename the `DMMeta` to `Meta` or `Metadata`?

Reply:
https://github.com/pingcap/tiup/pull/587/commits/9c6aae46dd607af798ffbf42e2e465a7b6c114b3